src/libmongoc/src/mongoc/mongoc-error.h

```diff
@@ -136,9 +137,10 @@ typedef enum {
 
    MONGOC_ERROR_CLIENT_INVALID_LOAD_BALANCER,
 
-   // Azure errors
-   MONGOC_ERROR_AZURE_HTTP,
-   MONGOC_ERROR_AZURE_BAD_JSON,
+   /* An error related to either GCP metadata or Azure IMDS server */
+   MONGOC_ERROR_KMS_SERVER_HTTP,
+   MONGOC_ERROR_KMS_SERVER_BAD_JSON,
```

Comment:
@kevinAlbs Do we want to keep the old names for these constants present? Have they been released for long enough to be relied on?

Reply:
No need. The constants were introduced in CDRIVER-4454 and have not been released yet.